code-challenges.js

```diff
@@ -39,3 +92,17 @@ var numbersToAdd2 = [0, 7, -8, 12]
 // Expected output: [0, 7, -1, 11]
 var numbersToAdd3 = []
 // Expected output: []
+
+const accumSum = (array) => {
+  return array.map((value, index) => {
+    if (index == 0){
+      return value
+    }
+    else{
+      return value + array[index-1]
+    }
+  })
+}
+console.log(accumSum(numbersToAdd1));
+console.log(accumSum(numbersToAdd2));
```

Comment:
the return values here are slightly off but I see where you were going! may just need some refactoring :)